### PR TITLE
feature/14-create-colors.scss file

### LINKED ISSUE #14: Create _colors.scss file
Create _colors.scss file which will store application colors definitions

1. Requirements
    - Create file _**_colors.scss**_ in _**assets**_ folder with name 
    - File should contain [scss variables](https://sass-lang.com/documentation/variables)

2. Example 
_colors.scss file content ⬇
`$pompeianRed: #a4292e`

More info in [Figma design](https://www.figma.com/file/BOSzvLT7foSt1jZprnNSbe/Angular-Material-(Material-3-Design-Component-Figma-Library-For-Angular)-(Community)?node-id=6%3A2)

### PR DESCRIPTION
**Pull request by DagmaraSkorupska**

### CODE REVIEW

**Review by jkbkupczyk — APPROVED** [review]
👍

**Comment by purbanski-deftcode** [comment]
I don't quire understand a purpose of this. In the project we are using Angular Material which can have theme configured for the whole application. This is where we should start I think. And add global custom colors whenever those will be required by some other tickets.

File: src/assets/styles/colors.scss [hunk 1]
```diff
@@ -1 +1,6 @@
-// define app colors
+$primary: #3F51B5;
+$white: #ffffff;
```
Comment (purbanski-deftcode) [inline src/assets/styles/colors.scss:2]: All colors are following uppercase convention but this one is not. Maybe you can unify this.
Also like I said in other comment I'm not sure if those changes are even needed currently.

File: src/assets/styles/colors.scss [hunk 2]
```diff
@@ -1 +1,6 @@
...
+$buttonConfirm: #82B1FF;
+$button: #3F51B5;
+$alertError: #FF5252;
```
Comment (purbanski-deftcode) [inline src/assets/styles/colors.scss:5]: There is `alert` or `error`, naming something `alertError` does not make much sense. Same for `successError` -> something can't be both a success and an error.

**Review by purbanski-deftcode — CHANGES_REQUESTED** [review]

**Pull request closed without merging**